src/main/java/com/linkedin/kmf/services/MultiClusterTopicManagementService.java

```diff
@@ -264,10 +274,14 @@ void maybeCreateTopic() throws Exception {
 
     private AdminClient constructAdminClient() {
       Properties adminClientProperties = new Properties();
-      adminClientProperties.setProperty(AdminClientConfig.SECURITY_PROTOCOL_CONFIG, SecurityProtocol.PLAINTEXT.name());
+      adminClientProperties.setProperty(AdminClientConfig.SECURITY_PROTOCOL_CONFIG, SecurityProtocol.SSL.name);
```

Comment:
This is still not what is needed.  Let's talk in person tomorrow.

Reply:
that's correct, the SSL portion is not committed yet. It is WIP.